btr-api/src/btr_api/resources/submission.py

```diff
@@ -110,6 +111,15 @@ def create_register():
         if not valid:
             return {"errors": errors}, HTTPStatus.BAD_REQUEST
 
+        # get entity
+        identifier = json_input['businessIdentifier']
+        entity = btr_entity.get_entity(jwt, identifier).json()
+
+        # validate entity; return BAD_REQUEST for historial companies and frozem company
+        entity_errors = validate_entity(entity)
+        if entity_errors:
+            return {"error": entity_errors}, HTTPStatus.BAD_REQUEST
```

Comment:
Sorry, I know I told you to use BAD_REQUEST for this, but I actually think it makes more sense to use FORBIDDEN for these entity rules.

Also can you please follow the same response format as `bad_request_response`, but with the forbidden code instead?